keyboards/syenakeyboards/elaruus/keyboard.json

```
@@ -0,0 +1,34 @@
+{
+    "manufacturer": "Syenasweta",
+    "keyboard_name": "SyenaKeyboards Elaruus",
+    "maintainer": "syenasweta",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": true,
+        "console": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
```

Unless there's a specific reason for `command`/`console` features to be enabled, they generally shouldn't be by default. I also don't reckon NKRO is necessary for this board.
```suggestion
        "extrakey": true,
        "mousekey": true
```